problems/2.functions.js

```diff
@@ -3,15 +3,15 @@
 * @returns {null}
 */
 function logGreeting() {
-
+  let name = "Hello Avery!"
```

Comment:
This will forever be "helo every" 
We need to use parameters. 
This function needs a console.log